test/spec/features/service/logs_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'spec_helper'
+
+describe 'service logs' do
+  before(:each) do
+    run("kontena service create --instances 2 test-1 redis:3.0")
+    run("kontena service deploy test-1")
+    sleep 1
+  end
+
+  after(:each) do
+    run("kontena service rm --force test-1")
+  end
+
+  it 'displays service logs' do
+    k = kommando("kontena service logs test-1")
+    expect(k.run).to be_truthy
+    expect(k.out.scan('The server is now ready to accept connections on port 6379').size).to eq(2)
```

Comment:
Scanning for the last line of logs output from the server startup is broken, because it will get stuck in the server logs buffer until some other action flushes it out:

```
test-1-2 | 1:M 07 Mar 09:00:13.961 * The server is now ready to accept connections on port 6379
weave | INFO: 2017/03/07 09:00:14.043612 [nameserver be:69:a5:ac:cb:73] adding entry for 9ac49592502d1503762f936957382438d372ca84d867e27bf5367279662b9f51: test-1-2.e2e.kontena.local. -> 10.81.128.89
test_api_1 | 172.19.0.1 - - [07/Mar/2017:09:00:18 +0000] "GET /v1/services/e2e/null/test-1/container_logs?limit=100 HTTP/1.1" 200 17818 0.5099
test_api_1 | 172.19.0.1 - - [07/Mar/2017:09:00:19 +0000] "DELETE /v1/services/e2e/null/test-1 HTTP/1.1" 200 2 0.0879
```

In this case, the output line is within 5 lines of the server `DELETE ...`, so no amount of sleep is really going to help.